### PR TITLE
Great Changes!

### LINKED ISSUE #12: Crseat's to do list
### To Do List

- [x] Create an Issue
- [x] Create a Branch
- [ ] Make changes and push
- [ ] Merge
- [ ] Have fun.

### PR DESCRIPTION
**Pull request by crseat**
# Some of the best code I've ever seen. 

### even more

fixes  #12

### CODE REVIEW

File: bestbranch.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+### Header
+
+Thing number 1
```
Comment (crseat) [inline bestbranch.md:3]: This is a great line right here.

File: bestbranch.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+Thing number 2 
+Thing number 3
+
+### HEader #2
```
Comment (crseat) [inline bestbranch.md]: Excellent header.

**Review by crseat — COMMENTED** [review]
No Explicit approval apparently!

**Review by zmsanche — CHANGES_REQUESTED** [review]
I would like to see more info about food.

File: bestbranch.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+### Header #2
+
+yaayy
+
+Here are some additional changes using the online editor. 
+
+I forgot to put a commit message in the last edit!
+
+More info about food for Zoë!!!
```
Comment (zmsanche) [inline bestbranch.md:15]: Wow this is some really good info @crseat

**Review by zmsanche — APPROVED** [review]

**Pull request merged**